src/assets/scripts/base/PositionModel.js

```diff
@@ -66,7 +66,7 @@ export default class PositionModel {
      * @param mode {string}                     Set to 'GPS' to indicate you are inputting lat/long that should
      *                                          be converted to positions
      */
-    constructor(coordinates = [], reference, magnetic_north = 0, mode) {
```

Comment:
It was still in use, but it didn't need to be. I've reworked the thing that was using it, so the mode is no longer needed. :smile: